plugin/trino-iceberg/src/test/java/io/trino/plugin/iceberg/TestIcebergStatistics.java

```diff
@@ -419,8 +419,8 @@ public void testAnalyzeSomeColumns()
                         VALUES
                           ('nationkey', null, 75, 0, null, '0', '74'),
                           ('regionkey', null, 15, 0, null, '0', '14'),
-                          ('comment', null, 50, 0, null, null, null), -- result of previous analyze
-                          ('name', null, 50, 0, null, null, null), -- result of previous analyze
+                          ('comment', null, null, 0, null, null, null), -- result of previous analyze
```

Comment:
This is not anymore the `result of previous analyze` right? 
`ANALYZE " + tableName + " WITH (columns = ARRAY['nationkey', 'regionkey'])` has replaced the stats for `comment` and `name` as well.
Does the comment still make sense?